src/PluginInfo/PluginInfo.js

```diff
@@ -149,9 +151,10 @@ function PluginInfo (dirname) {
 
     function _parseEditConfigs (tag) {
         var editConfig =
-            { file: tag.attrib['file'],
-                target: tag.attrib['target'],
-                mode: tag.attrib['mode'],
+            {
```

Comment:
Can we bring this up to the assignment line and unindent the object body please?